compose/config/config.py

```diff
@@ -1154,6 +1154,20 @@ def merge_deploy(base, override):
     return dict(md)
 
 
+def merge_networks(base, override):
+    merged_networks = {}
+    all_network_names = set(base) | set(override)
+    base = {k: {} for k in base} if isinstance(base, list) else base
+    override = {k: {} for k in override} if isinstance(override, list) else override
+    for network_name in all_network_names:
+        md = MergeDict(base.get(network_name, {}), override.get(network_name, {}))
+        md.merge_scalar('aliases')
+        md.merge_scalar('ipv4_address')
+        md.merge_scalar('ipv6_address')
+        merged_networks[network_name] = dict(md)
```

Comment:
Missing [`linked_local_ips`](https://docs.docker.com/compose/compose-file/compose-file-v2/#link_local_ips) (merge strategy same as aliases) and [`priority`](https://docs.docker.com/compose/compose-file/compose-file-v2/#priority) (simple `merge_scalar` as it's just an int value)

Reply:
Adding